libs/core/src/lib/card/kpi/card-kpi-value.directive.ts

```diff
@@ -23,7 +29,7 @@ export class CardKpiValueDirective implements OnInit, CssClassBuilder {
     @applyCssClass
     /** @hidden */
     buildComponentCssClass(): string[] {
-        return [CLASS_NAME.cardAnalyticsKpiValue];
+        return [CLASS_NAME.cardAnalyticsKpiValue, this.status ? `fd-numeric-content__kpi--${this.status}` : ''];
```

Comment:
Numeric content is a separate component that is not connected to Card component. It can be used to construct an analytical card but it can also be part of any other component. I don't understand why it's part of Cards.